### PR TITLE
Snapshot hint

### LINKED ISSUE #1179: Expect_snapshot_file() hint is not very helpful in R CMD check
Because when such a test fails in R CMD check, one cannot run snapshot_review()?

### PR DESCRIPTION
**Pull request by maelle**
Fix #1179

### CODE REVIEW

**Comment by hadley** [comment]
@maelle could you please take a look at the update language/approach?

**Comment by maelle** [comment]
It looks great 😀

File: R/snapshot-file.R [hunk 1]
```diff
@@ -80,6 +80,16 @@ expect_snapshot_file <- function(path, name = basename(path), binary = TRUE, cra
   )
 }
 
+snapshot_hint <- function(file, ci = on_ci(), check = in_rcmd_check()) {
+  paste0(
+    if (check && ci) "* Download run artifact to retrieve snapshot differences\n",
+    if (check && !ci) "* Locate check directory\n",
+    if (check) "* Copy 'new' snapshot to local test directory\n",
```
Comment (maelle) [inline R/snapshot-file.R:87]: This could even be more specific? I.e. the snapshot folder

Comment (maelle) [reply R/snapshot-file.R:87]: Even the filename?

**Pull request merged**